build/mvn

```diff
@@ -68,7 +68,7 @@ install_app() {
 # Install maven under the build/ folder
 install_mvn() {
   install_app \
-    "http://apache.claz.org/maven/maven-3/3.2.3/binaries" \
+    "http://archive.apache.org/dist/maven/maven-3/3.2.3/binaries" \
```

Comment:
While you're at it, point to 3.2.5? It's not ideal to point to `archive.apache.org` but at least it holds releases longer, whereas mirrors drop all but the latest point release in each line. I don't know of a redirector that would automatically send the request to a mirror though.